_pages/getting-started/getting-started.md

```diff
@@ -153,9 +153,7 @@ especially if you haven't worked on information security before. Read the ["Secu
 If you're looking to go into greater detail,
 18F's [Before You Ship](https://before-you-ship.18f.gov/) guide is definitive and should answer any questions you have,
 but it can also be a little overwhelming.
-If you ever need a quick answer, clarification or just general Security and Compliance help,
-head over to [#wg-security](https://gsa-tts.slack.com/messages/CG8SNAEQM)
-or [#wg-compliance](https://gsa-tts.slack.com/messages/C2UUQ5CNP) in Slack.
+If you ever need a quick answer, clarification or just general Security and Compliance help, fill out the [TTS Tech Portfolio General Request Form](https://forms.gle/wPrCXJqDgaksXRwV8) and selecting the option for a _New System (ATO/Architecture consult)_.
```

Comment:
I don't think this is a good spot, either. I think directing them to slack channels is a better first stop. It's why they exist.

Reply:
What about keeping both?

I don't think I knew TTS had a standard that slack should be the first and any form second or that was one of the purpose of slack's existence. I think we should keep that in mind when working on https://github.com/18F/tts-tech-portfolio/issues/249

Reply:
I think there's a cultural value, more than a standard, to value the quick answer from just communicating with other humans over a form. Or, to put it another way, why fill out the form for something that's just a quick slack question? It spreads the load of answering, and  since Slack is public, the answer or clarifcation is public and searchable.